content/documentation/staging/architecture.adoc

```diff
@@ -67,15 +64,16 @@ data is stored in Prometheus. Kiali uses the data stored in Prometheus to
 figure out the mesh topology, show metrics, calculate health, show possible
 problems, etc.
 
-Kiali communicates directly with Prometheus and assumes the data schema used by
-Istio Telemetery. It's a hard dependency for Kiali, and most of it's
-features do not work without it.
+Kiali communicates directly with Prometheus and assumes the metrics used by
+Istio Telemetery. It's a hard dependency for Kiali, and many Kiali features
+will not work without it.
 
 Currently, Kiali relies on
 https://istio.io/docs/reference/config/policy-and-telemetry/metrics/[Istio's
 default metrics] set. Make sure that these default metrics are always in place.
-Custom metrics are not supported by Kiali. If you need custom metrics or a
-variant of the default ones, create new Istio metrics with custom names.
+Some metric customization is possible as long the Kiali requirements are
+still met.  For the current list of required metrics see
+link:https://kiali.io/documentation/latest/faq/#prom-metrics[this FAQ entry].
```

Comment:
Minor, but I prefer relative links. Probably this may work: `link:{{< relref "./faq#prom-metrics" >}}`.

It makes it a little easier to review, because when clicking the link you don't land in the real website :)

Anyway, not a blocker. We can merge without a change.

Reply:
I'll try the relative link, I didn't know the syntax, thanks!